src/basic/smack-util.c

```diff
@@ -202,7 +181,37 @@ int mac_smack_fix(const char *path, bool ignore_enoent, bool ignore_erofs) {
 
                 r = log_debug_errno(errno, "Unable to fix SMACK label of %s: %m", path);
         }
-#endif
 
         return r;
 }
+
+
+#else
+bool mac_smack_use(void) {
+        return false;
+}
+
+int mac_smack_read(const char *path, SmackAttr attr, const char **label) {
+        return 0;
+}
+
+int mac_smack_read_fd(int fd, SmackAttr attr, const char **label) {
+        return 0;
+}
+
+int mac_smack_apply(const char *path, SmackAttr attr, const char *label) {
+        return 0;
+}
+
+int mac_smack_apply_fd(int fd, SmackAttr attr, const char *label) {
+        return 0;
+}
+
+int mac_smack_apply_pid(pid_t pid, const char *label) {
+        return 0;
+}
+
+int mac_smack_fix(const char *path, bool ignore_enoent, bool ignore_erofs) {
+        return 0;
+}
```

Comment:
I guess these functions could really return something like `-EOPNOTSUPP`, because they shouldn't be called when `mac_smack_use()` return `false`, right?